examples/label_export/video.ipynb

```diff
@@ -2,22 +2,17 @@
   "cells": [
```

Comment:
<p>is it helpful to show this, or can we leave it up to the user to run the code block to see it themselves? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>